src/raw_client.rs

```diff
@@ -381,6 +381,7 @@ impl RawClient<ElectrumSslStream> {
     ) -> Result<Self, Error> {
         use std::convert::TryFrom;
 
+        let _ = rustls::crypto::aws_lc_rs::default_provider().install_default();
```

Comment:
I believe this should be behind feature gate:
```suggestion
        #[cfg(feature = "use-rustls")]
        let _ = rustls::crypto::aws_lc_rs::default_provider().install_default();
```

Reply:
fixed